google/cloud/bigtable/async_row_reader_test.cc

```diff
@@ -69,13 +69,13 @@ class TableAsyncReadRowsTest : public bigtable::testing::TableTestFixture {
             std::move(request_expectations));
 
     EXPECT_CALL(*client_, PrepareAsyncReadRows)
-        .WillOnce([&reader, request_expectations_ptr](
+        .WillOnce([this, &reader, request_expectations_ptr](
                       grpc::ClientContext* context,
                       btproto::ReadRowsRequest const& r,
                       grpc::CompletionQueue*) {
-          EXPECT_STATUS_OK(
-              IsContextMDValid(*context, "google.bigtable.v2.Bigtable.ReadRows",
-                               google::cloud::internal::ApiClientHeader()));
+          EXPECT_STATUS_OK(validate_metadata_fixture_.IsContextMDValid(
```

Comment:
there is only one call. Maybe we just define `ValidateMetadataFixture fixture;` in here?

Reply:
The function around this is called many times though, I am happy with it as-is.